client/src/components/Applications/AppForm/AppFormUtils.js

```diff
@@ -2,8 +2,13 @@ import * as Yup from 'yup';
 import { getSystemName } from 'utils/systems';
 
 export const TARGET_PATH_FIELD_PREFIX = '_TargetPath_';
+export const DEFAULT_JOB_MAX_MINUTES = 60 * 24;
 
 export const getQueueMaxMinutes = (app, queueName) => {
+  if (!isJobTypeBATCH(app)) {
```

Comment:
This is still needed because of some logic in help text which tries to get max minutes based on queue.